rclcpp/include/rclcpp/node.hpp

```diff
@@ -1072,6 +1072,58 @@ class Node : public std::enable_shared_from_this<Node>
   bool
   assert_liveliness() const;
 
+  /// Return a list of publishers about topic endpoint information to a given topic.
+  /**
+   * The returned parameter is a list of topic endpoint information, where each item will contain
+   * the node name, node namespace, topic type, endpoint type, participant's GID and its QoS
```

Comment:
Same as in `rclpy`, `participant` is not a ROS term, please limit documentation to ROS terminology.

Reply:
Switched over to using `topic endpoint`.